exchanges/binance/binance.go

```diff
@@ -603,17 +604,36 @@ func (b *Binance) SendAuthHTTPRequest(method, path string, params url.Values, re
 	signature := params.Encode()
 	hmacSigned := common.GetHMAC(common.HashSHA256, []byte(signature), []byte(b.APISecret))
 	hmacSignedStr := common.HexEncodeToString(hmacSigned)
-	params.Set("signature", hmacSignedStr)
 
 	headers := make(map[string]string)
 	headers["X-MBX-APIKEY"] = b.APIKey
 
 	if b.Verbose {
-		log.Debugf("sent path: \n%s\n", path)
+		log.Debugf("sent path: %s", path)
 	}
+
 	path = common.EncodeURLValues(path, params)
+	path += fmt.Sprintf("&signature=%s", hmacSignedStr)
```

Comment:
Guessing this is because alphabetical sorting breaks the request?

Reply:
That's correct

Reply:
Signature needs to be put at the end.